### ISSUE TITLE
Cannot download testing data

### ISSUE THREAD

**Issue by 601SunG** [hjkuijf/wmhchallenge#4]
Hi, 
I was intend to use the data from challenge as my testing data, but when I log in and try to download the data, only training data (60 subjects) is available, would you please help me to know where could I find the corresponding testing data? 

Many thanks

**Comment by hjkuijf** [comment]
The test data is not available. The challenge is still ongoing and you can submit your method for evaluation on the test set. The organizers will then run your method on the test data, as explained here: https://wmh.isi.uu.nl/methods/submit/ . Note that the agreement does not allow to use the training data as a test.
